### ISSUE TITLE
Problems with botanical legacy rank 'prole(s)'

### ISSUE THREAD

**Issue by aoern** [CatalogueOfLife/data#510]
In the vocabulary of COLDP, there is a definition of rank 'prole' as follows:
{"ambiguous":false,"familyGroup":false,"genusGroup":false,"infraspecific":true,"legacy":true,
"linnean":false,"majorRank":"infraspecific name","marker":"prol.","name":"prole","plural":"proles","restrictedToCode":"botanical","suprageneric":false,"supraspecific":false,"uncomparable":false}
This definition is incorrect: There is no such rank as 'prole', the correct rank name is 'proles' (race in English). The plural form of 'proles' is 'proles', too.
This error seems to cause 652 erroneous rank names 'prole' in the database, for example
Agrostis alba prol. prorepens Rouy.
There are also 468 cases where the name parser is confused by the 'proles' in the scientific  name, for example:
'Capparis spinosa proles' (no authorship) is introduced as a synonym of Capparis spinosa subsp. rupestris (Sm.) Nyman. World Plants displays the name as "Capparis spinosa ¿proles rupestris (Sm.) Rouy & Foucaud".

**Comment by mdoering** [comment]
rank enum changed in prod, so you get this now:
<img width="1388" alt="image" src="https://user-images.githubusercontent.com/327505/220350840-23362885-ca94-4ae7-9c1a-95c91ca3b1da.png">

**Comment by yroskov** [comment]
@mdoering, do I need to take any action on my side?

**Comment by mdoering** [comment]
No, but you might want to verify it is indeed all solved and then close this

**Comment by aoern** [comment]
Situation in Mar 2023 release:
1. The rank of Agrostis alba prol. prorepens Rouy  is now proles. OK.
2. Misparsing "Capparis spinosa ¿proles rupestris (Sm.) Rouy & Foucaud" to "Capparis spinosa proles (no authorship)" still happens. Not OK.

**Comment by aoern** [comment]
P.S.
See also synonym of Juncus capitatus: "Juncus capitatus unranked" vs.
"Juncus capitatus ¿unranked versicolor Asch. & Graebn."

**Comment by mdoering** [comment]
@yroskov I dont know how frequent the usage of `¿` is, but it might warrant an update decision...

**Comment by yroskov** [comment]
¿ might be present in WPlants/Ferns dataset as a technical tag. Michael promised to remove them before he send data to CoL.
(@gdower, would it be possible to check Micahel's data for remaining ¿, and clean up them during conversion?)
